### PR TITLE
[docs] Update quick start

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- Update quick start docs (staking)

### CODE REVIEW

File: ain-blockchain/developer-guide/getting-started.md [hunk 1]
```diff
@@ -255,9 +272,40 @@ ain.db.ref(appPath).getOwner()
     */
   });
 ```
+
+{% endcode %}
+
+## Step 8. Stake AIN to your app
```
Comment (jiyoung-an) [inline ain-blockchain/developer-guide/getting-started.md:278]: I added staking step (cc: @platfowner @yoojinko)

File: ain-blockchain/developer-guide/getting-started.md [hunk 2]
```diff
@@ -22,23 +22,29 @@ npm install @ainblockchain/ain-js
...
+Chain ID must be set to 1 to use [ain-js](https://github.com/ainblockchain/ain-js) on the Mainnet. (0: Testnet, 1: Mainnet)
 
 {% code title="transfer.js" %}
+
 ```javascript
 const Ain = require('@ainblockchain/ain-js').default;
-const ain = new Ain('https://testnet-api.ainetwork.ai', 0);
+const ain = new Ain('https://testnet-api.ainetwork.ai', null, 0);
+
+// To use the Mainnet, you need to initialize as follows:
```
Comment (jiyoung-an) [inline ain-blockchain/developer-guide/getting-started.md:33]: I added initialization code for mainnet (cc: @platfowner)

File: ain-blockchain/developer-guide/getting-started.md [hunk 3]
```diff
@@ -22,23 +22,29 @@ npm install @ainblockchain/ain-js
...
+// const ain = new Ain('https://mainnet-api.ainetwork.ai', null, 1);
```
Comment (jiyoung-an) [inline ain-blockchain/developer-guide/getting-started.md]: I added initialization code for mainnet (cc: @platfowner)

File: ain-blockchain/developer-guide/getting-started.md [hunk 4]
```diff
@@ -255,9 +272,40 @@ ain.db.ref(appPath).getOwner()
...
+```javascript
+const Ain = require('@ainblockchain/ain-js').default;
+const ain = new Ain('https://testnet-api.ainetwork.ai', null, 0);
+
+// Import the account you've created in Step 3.
+ain.wallet.addAndSetDefaultAccount(YOUR_PRIVATE_KEY);
+const myAddress = ain.wallet.defaultAccount.address;
+
+const appName = 'my_bot'; // Use your own app name
+const appPath = `/apps/${appName}`;
```
Comment (yoojinko) [inline ain-blockchain/developer-guide/getting-started.md]: app path 안쓰이는 것 같습니다

**Review by yoojinko — APPROVED** [review]
LGTM with a comment. Thanks!

File: ain-blockchain/developer-guide/getting-started.md (hunk 4 above)
Comment (jiyoung-an) [reply ain-blockchain/developer-guide/getting-started.md]: done!

**Comment by jiyoung-an** [comment]
Thank you for review @yoojinko

File: ain-blockchain/developer-guide/getting-started.md (hunk 1 above)
Comment (platfowner) [reply ain-blockchain/developer-guide/getting-started.md:278]: thanks a lot!

**Review by platfowner — COMMENTED** [review]
LGTM
Thanks!

**Pull request merged**